.github/scripts/close_nonexistent_disable_issues.py

```diff
@@ -3,54 +3,67 @@
 import multiprocessing as mp
 import os
 import re
+import sys
 import tempfile
-from typing import Any, Dict, List, Optional, Tuple
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
 
 import requests
-import rockset  # type: ignore[import]
 from gitutils import retries_decorator
 
 
+REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+sys.path.insert(0, str(REPO_ROOT))
+from tools.testing.clickhouse import query_clickhouse
+
+
+sys.path.pop(0)
+
+
 LOGS_QUERY = """
 with
     shas as (
         SELECT
-            push.head_commit.id as sha,
+            distinct
+            push.head_commit.id as sha
         FROM
-            commons.push
+            -- Not bothering with final here
+            default.push
         WHERE
             push.ref = 'refs/heads/viable/strict'
-            AND push.repository.full_name = 'pytorch/pytorch'
+            AND push.repository.'full_name' = 'pytorch/pytorch'
         ORDER BY
-            push._event_time DESC
+            push.head_commit.'timestamp' desc
         LIMIT
             5
     )
 select
     id,
     name
 from
-    workflow_job j
+    default.workflow_job j final
     join shas on shas.sha = j.head_sha
 where
-    j.name like '% / test%'
+    j.id in (select id from materialized_views.workflow_job_by_head_sha where head_sha in (select sha from shas))
+    and j.name like '% / test%'
     and j.name not like '%rerun_disabled_tests%'
     and j.name not like '%mem_leak_check%'
 """
 
 TEST_EXISTS_QUERY = """
 select
-    count(*) as c
+    name
```

Comment:
Nit: Why not just count it here like the original query? (I couldn't find what is that name is used for, so I guess it's not needed?)

Reply:
It's faster to just find one than to find all of them